src/components/TopicItem/index.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { TouchableOpacity, Image, Text } from 'react-native';
+import { number, string, func } from 'prop-types';
+import styles from './styles';
+
+const TopicItem = ({ id, label, icon, onSelect }) => {
+  return (
```

Comment:
```suggestion
const TopicItem = ({ id, label, icon, onSelect }) => (
```
You can return directly here